### ISSUE TITLE
Missing Owner Org when harvesting RDF source

### ISSUE THREAD

**Issue by Hoedic** [ckan/ckanext-dcat#26]
While running an import from a DCAT RDF source, the harvester tells the owner org is missing:

```2015-03-26 01:54:24,757 INFO  [ckanext.harvest.queue] Received harvest object id: 68d9d538-313c-40f8-83e4-711bcc91afd7
2015-03-26 01:54:24,793 DEBUG [ckanext.dcat.harvesters.rdf] In DCATRDFHarvester import_stage
2015-03-26 01:54:24,836 DEBUG [ckanext.harvest.harvesters.base] Create validation Error: {'Owner org': 'Missing value'}```

I am trying to import the RDF example : https://github.com/ckan/ckanext-dcat/blob/master/examples/catalog_datasets_list.rdf

When configuring the harvester, I used the field "organization" in the for to specify an organization in CKAN. At some point I also tried to put owner_org: my-organization in the configuration textarea.

now my configuration fields look like that (I don't know if it's of any use...):
```{
 "default_groups":["a-group"],
 "user":"admin",
 "read_only": false
} ```

After looking at the code, I don't see any place where the fetch process tries to read the configuration.

**Comment by amercader** [comment]
You are correct @Hoedic, the harvester does not handle owner organizations and that will fail if your CKAN instance requires datasets to have one. Can you try this branch and see if it solves the issue:

https://github.com/ckan/ckanext-dcat/pull/27

It should set the harvested datasets to the same organization that the harvest source belongs to (the one you set when creating the harvest source)

**Comment by amercader** [comment]
You are correct @Hoedic, the harvester does not handle owner organizations and that will fail if your CKAN instance requires datasets to have one. Can you try this branch and see if it solves the issue:

https://github.com/ckan/ckanext-dcat/pull/27

It should set the harvested datasets to the same organization that the harvest source belongs to (the one you set when creating the harvest source)

**Comment by lubertorubior** [comment]
Hi, could this also be done for a "DCAT JSON Harvester" source? 
The configuration of the source doesn't seem to be read.

**Comment by amercader** [comment]
@lubertorubior I've pushed changes to support the JSON harvester as well. Can you test the branch on #27 to see if it works as expected?

**Comment by lubertorubior** [comment]
It works great, thank you @amercader !

**Comment by amercader** [comment]
Fixed in #27
